src/dbl.ml

```diff
@@ -47,6 +47,10 @@ let cmd_args_options = Arg.align
     Arg.String (fun p -> cli_local_search_dirs := p :: !cli_local_search_dirs),
     " Add a path to local search directories";
 
+    "--no-error-context",
```

Comment:
One minor thing: in `Arg` library, as well as in other OCaml tools, flags starts with single dash. I'm not happy with this practice, but since we use this library, we should respect it. I'll fix it before merge.